ts/features/profile/components/Error.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { H3, Pictogram, VSpacer } from "@pagopa/io-app-design-system";
+import { View } from "react-native";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+
+export const ErrorComponent = () => (
+  <View style={[IOStyles.flex, IOStyles.centerJustified, IOStyles.alignCenter]}>
```

Comment:
What do you think to use a `SafeAreaView` and a `ContentWrapper`? https://github.com/pagopa/io-app-design-system/blob/main/src/components/contentWrapper/ContentWrapper.tsx